lib/board.rb

```diff
@@ -0,0 +1,94 @@
+require 'pry'
+
+class Board
+    attr_accessor :cells, :token
+    
+    def initialize
+        
+        @cells = [" "," "," "," "," "," "," "," "," "]
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.